_posts/2024-12-19-cosmicqc.md

```diff
@@ -0,0 +1,313 @@
+This is also an important step in not just single-cell, but also bulk RNA-seq as well, which has existed for over two decades.
+scRNA-seq has been around for over a decade, so there has been a lot of time to develop these methods across both technologies.
+
+[Cell Painting](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0080999) was first developed in 2013, meaning that image-based profiling and extraction of morphological profiles is a new field, but it still has been around for [over a decade](https://arxiv.org/abs/2405.02767).
+However, single-cell quality control for morphology profiling is not nearly as thoroughly developed. 😢
+
+This lack of development could be because of the lack of trust in this method, which is described in the 2017 paper, "[Data-analysis strategies for image-based cell profiling](https://www.nature.com/articles/nmeth.4397)".
+Caicedo et al. describes some methods of cell-level outlier detection, including model-free and model-based, but not specifically related to image-based profiling.
+Their main concern is that this method of quality control can assume homogeneity and remove interesting phenotypes (spoiler, we do run into this issue!).
+This is why most labs skip this step.
+Honestly, this concern applies to all quality control methods and underscores the importance of striving for continuous improvement.
+
+Currently, you are able to filter objects in CellProfiler using morphology measurements in its own module.
+You can set limits for what feature measurement is considered good or bad, but without exporting the data and exploring the feature space, you cannot know the range of values for your dataset.
+In my opinion, this module is good to have, but it isn't a solution to single-cell quality control.
+
+I checked to see if there were existing single-cell quality control methods for image-based profiling.
+I was only able to find one paper, published in BMC Bioinformatics, called "[A cell-level quality control workflow for high-throughput image analysis](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-020-03603-5)", and it was published in 2020.
+It was an interesting read!
+
+For example, in the background section, Qiu et al. state:
+
+> *"To our knowledge, this represents the first QC method and scoring metric that operate at the cell level."*
+
+Since I couldn't find any other paper that focuses on quality control at the single-cell level published before, I completely agree with their statement, they are definitely the first.
+
+I actually did come across another paper after this one (published in 2022) that cites their work, called "[Image-based cell profiling enhancement via data cleaning methods](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0267280)"
+I won't be going into depth in this blog post about this paper, but the main highlight is that they did make an attempt at cell-level outlier detection.
```

Comment:
```suggestion
The main highlight in this paper, is that they did attempt cell-level outlier detection.
```

Reply:
Let's plan on doing a JC of this article in 2025!

Reply:
💯 Absolutely! I can plan that for my first one in Feb (based on the schedule)